tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, name, dynamodb_region, stopping=False) -> None:
         self.dynamodb_region = dynamodb_region
         self.table = self.dynamodb.Table(name)
         self.stopping = stopping
-        self.save_queue = OrderedDict()
+        self.save_queue = OrderedDict()  # type: OrderedDict
```

Comment:
we should be able to type this as `self.save_queue: OrderedDict = OrderedDict()`

that said, not sure why mypy is suddenly unhappy here